content/desktop/release-notes.md

```diff
@@ -24,11 +24,71 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
-## 4.24.2
+## 4.25.0
+
+{{< release-date date="2023-10-26" >}}
+
+{{< desktop-install all=true version="4.25.0" build_path="/" >}}
+
+### Upgrades
+
+- [Compose v2.23.0](https://github.com/docker/compose/releases/tag/v2.23.0)
+- [Docker Scout CLI v1.0.9](https://github.com/docker/scout-cli/releases/tag/v1.0.9).
+- [Kubernetes v1.28.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.28.2)
+- [cri-dockerd v0.3.4](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.3.4)
+- [CNI plugins v1.3.0](https://github.com/containernetworking/plugins/releases/tag/v1.3.0)
+- [cri-tools v1.28.0](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.28.0)
+### New
+
+- Rosetta is now available for all users on MacOS 13 or greater, providing faster emulation of Intel-based images on Apple Silicon.
+- Docker Desktop will now detect if a WSL version is out of date. If an out dated version of WSL is detected, you can now optionally allow Docker Desktop automatically updates the installation or manually update WSL outside of Docker Desktop.
+- Installation pre-requisites for Windows is now higher than 19043 for both GAC and LTSC.
```

Comment:
Will people know what GAC and LTSC is? Is it necessary? We don't use this in our installation pages

Reply:
yeah, it's a bit to obscure maybe. I think there is a version of 19043 that still has support, @KatTomrushka might have more insight on that. We can leave it at 

`- Installation pre-requisites for Windows is now higher than 19043` 

?

Reply:
Hey both, it's based on the Windows channels wiki: https://en.wikipedia.org/wiki/Windows_10_version_history#Channels

I have also prepared a more user friendly version that is going into the blog: https://docs.google.com/document/d/1bxQCB4jqSN-Fx0yFraFrX0jNS3tefhoa6PVFeZzidrU/edit#heading=h.1hp66jvcc8ul

Reply:
Thanks @KatTomrushka for the extra context, I think given the blog info and the rest of docs, your second suggestion is fine @lorenrh. Let's keep it consistent with how we present it elsewhere

Reply:
actually let's go with `Docker Desktop for Windows now requires a Windows version of 19044 or later.`